Source/Blazorise.AntDesign/Select.razor.cs

```diff
@@ -315,7 +329,7 @@ string DropdownClassNames
     }
 
     string DropdownStyleNames =>
-        $"width: {(int)elementInfo.BoundingClientRect.Width}px; left: {(int)elementInfo.OffsetLeft}px; top: {(int)( elementInfo.OffsetTop + elementInfo.BoundingClientRect.Height )}px;";
+        $"width: {(int)dropdownElementInfo.BoundingClientRect.Width}px; left: {(int)dropdownElementInfo.OffsetLeft}px; top: {(int)( dropdownElementInfo.OffsetTop + dropdownElementInfo.BoundingClientRect.Height )}px;";
```

Comment:
I still don't get, how this is different then before? Maybe I'm not seeing or understanding something, can you explain?

Reply:
Thing is I tried to make select list in antD to ref on dropdownElement not on element due to it sets wrong position for that list.

Reply:
Basicalyit gets wrong info and sets wrong element.styles

Reply:
I still do not understand... :( 
Was this an attempt that did not work? Or is this working? Because personally, with the current code I do not see a difference between elementInfo and dropdownElementInfo

Is this PR working as expected? Or are we still figuring out an issue? We've marked this PR as review ready.

Reply:
I'm just trying to fix it, this doesn't work, but I'm trying to find a solution to fix the positioning of the select list, which for some reason appears in the wrong position and has the wrong width

Reply:
when I was fixing it, I noticed that when moving the position of the absolute from the class (ant-selecat dropdown), the list moves to the right position, but it breaks some other elements in blazorise that are directly connected to that same class, now I'm trying to find another way because it seems that this is not a good approach either

Reply:
Ah gotcha! No point in reviewing not ready code. Solution might change completly.
I'll see if I can take a look later today if you'd like.

Reply:
yes, why not, I'll try to find a solution, so if I change anything, I'll push the changes so you can take a look